stdlib/public/Cxx/std/String.swift

```diff
@@ -25,6 +25,12 @@ extension std.string: ExpressibleByStringLiteral {
   }
 }
 
+extension std.string: CustomDebugStringConvertible {
```

Comment:
Any reason not to _also_ add a `CustomStringConvertible` conformance?

```swift
extension std.string: CustomStringConvertible {
  public var description: String {
    String(self)
  }
}
```

Unfortunately, once we ship these types, it seems unlikely we'll ever be able to add any new conformances to them to pre-existing protocols, so it might be wise to consider doing this for _all_ newly imported types, as a matter of course.

The same gotcha applies to `Equatable`, `Comparable` and `Hashable` -- if there is a nonzero chance we want to make any these C++ types conform to those later, then we need to add those conformances up front. (I have _immense_ regrets about some basic stdlib types not conforming to these.)

Reply:
Good to know, it does seem like I good idea to have 'CustomStringConvertible' conformance too. I'll create another PR to add that conformance in.